src/css/dev.css

```diff
@@ -0,0 +1,10 @@
+/** Source Sans 3, Source Serif 4, Roboto Slab, Roboto Mono (regular only) */
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono&family=Roboto+Slab:wght@300;400;700&family=Source+Sans+3:ital,wght@0,400;0,600;0,700;1,400;1,600;1,700&family=Source+Serif+4:ital,wght@0,400;0,600;0,700;1,400;1,600;1,700&display=swap');
+
+@import 'tailwindcss';
+@source not '../../docs';
```

Comment:
Huh. Funny syntax. [docs](https://tailwindcss.com/docs/detecting-classes-in-source-files#ignoring-specific-paths)

Reply:
Some of these things were hard to find 😬